### PR TITLE
Feature/universal wheel

### PR DESCRIPTION
**Pull request by fredrikaverpil**
Referring back to issue #162 - here's how we could avoid wheel fragmentation of Qt.py since it's a pure Python module.

### CODE REVIEW

**Comment by mottosso** [comment]
Ok, so the version picked up by PyPI (the irreversible and forcible unique identifier) is the one from `setup.py`.

As we test, we'll (1) set this manually to something temporary that we can increment, like `0.6.7-test-01` and (2) merge it, (3) make a release, called the same as the temporary version. Once that's done, Travis will attempt to automatically upload it to PyPI, and from there we can install it and see whether it works.

It's a bit of a terrible way of testing, but I can't see any other way. Once we've merged a couple of times and we see that it's working, we can remove the tags from the repo, along with the releases on GitHub and set the version back to something sensible.

File: Qt.py [hunk 1]
```diff
@@ -69,7 +69,7 @@
 self.__modified__ = list()  # Existing members modified in some way
 
 # Below members are set dynamically on import relative the original binding.
-self.__version__ = "0.6.6"
+self.__version__ = "0.6.7-test-01"
```
Comment (mottosso) [inline Qt.py]: You can set it here, or in setup.py. Either is ok.

Comment (fredrikaverpil) [inline Qt.py]: Yup, it was done in 15a6eab. Looks ok?

**Comment by fredrikaverpil** [comment]
> It's a bit of a terrible way of testing

Indeed, but I don't see any other way either.

So, are we ready to merge?
Would you like to do the honors? 😄

**Comment by mottosso** [comment]
It's all yours. :)

**Pull request merged**